services/collaboration/pkg/middleware/tracing.go

```diff
@@ -0,0 +1,40 @@
+package middleware
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func CollaborationTracingMiddleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		wopiContext, err := WopiContextFromCtx(r.Context())
+		if err != nil {
+			// if we can't get the context, skip this middleware
+			next.ServeHTTP(w, r)
```

Comment:
Should we log this somehow? Or we simply don't care?

Reply:
If it's set properly, I don't think we'll reach that line. If there isn't a valid WOPI context, the `WopiContextAuthMiddleware` should have failed the request before reaching the tracing middleware.